roqoqo-qryd/src/tweezer_devices.rs

```diff
@@ -900,7 +915,7 @@ impl TweezerDevice {
             })?;
 
         let mut names: HashSet<&str> = HashSet::new();
-        if let Some(info) = self.layout_register.get(&layout_name) {
+        if let Some(info) = self.layout_register.as_ref().unwrap().get(&layout_name) {
```

Comment:
The unwraps here might be dangerous. If I understand correctly they should panic whenever the TweezerDevice represents an emulator device which could happen when pulling a device from the API with the corresponding name. Maybe this should be replaced with returning a descriptive error.